scripts/zones/Mamook/mobs/Gulool_Ja_Ja.lua

```diff
@@ -4,31 +4,63 @@
 -----------------------------------
 
 require("scripts/globals/titles");
-
------------------------------------
--- onMobSpawn Action
------------------------------------
+require("scripts/globals/status");
+require("scripts/zones/Mamook/MobIDs");
 
 function onMobSpawn(mob)
+    mob:setLocalVar("1Hour", math.random(3,5)); 
```

Comment:
there's now a mixin you can use to give a mob its 1hr/job special, so this line and the code block in onMobFight aren't needed.  See
https://github.com/DarkstarProject/darkstar/blob/e46bce5064fa0631c4a17b34837bea4ecb50aa81/scripts/zones/Lower_Delkfutts_Tower/mobs/Epialtes.lua#L5